src/nvim/tui/tui.c

```diff
@@ -423,9 +426,16 @@ static void tui_resize(UI *ui, int width, int height)
   ugrid_resize(&data->grid, width, height);
 
   if (!got_winch) {  // Try to resize the terminal window.
-    char r[16];  // enough for 9999x9999
-    snprintf(r, sizeof(r), "\x1b[8;%d;%dt", height, width);
-    out(ui, r, strlen(r));
+    // Only send this control sequence extension to terminal types that we know understand it.
```

Comment:
[The linter](https://travis-ci.org/neovim/neovim/jobs/228865508#L2992) complains about line length here and a couple other lines to line 434...if it is possible to shorten anything while retaining the meaning that would be swell - no big deal though.